### DISCUSSION TITLE
Your GitHub action to install LLVM

### DISCUSSION THREAD

**Discussion by thi8v** [mun-lang/mun#525] [Q&A] [answered]
Why you don't release your Github Action for Installing LLVM. I think I could use it for my project and everyone who wants to install LLVM in a github workflow could love your action.

**Comment by Wodann** [comment, marked as answer]
That's a great idea!

I've created a ["good first issue"](https://github.com/mun-lang/mun/issues/526) to do that. We'd very much appreciate your help with this! Alternatively, we or one of our community contributors will get around to it when they have time 🙂
